src/screens/AudioScreen/Audio.tsx

```diff
@@ -1,45 +1,41 @@
-import { Text, View } from "react-native";
-import ViewContainer from "../../components/ViewContainer";
-import RectButton from "../../components/RectButton";
-import globalStyles from "../../globalStyles";
-import styles from "./styles";
+import { Text } from 'react-native';
+import { useTranslation } from 'react-i18next';
+import ViewContainer from '../../components/ViewContainer';
+import RectButton from '../../components/RectButton';
+import globalStyles from '../../globalStyles';
+import styles from './styles';
+import { namespaces } from '../../i18n/i18n.constants';
+import { RootStackScreenProps } from '../../types/navigation';
+import '../../i18n/i18n';
 
-import { useTranslation } from "react-i18next";
-import { namespaces } from "../../i18n/i18n.constants";
-import "../../i18n/i18n";
-import React, { useState, useEffect } from "react";
-{
-  "compilerOptions"; {
-    "allowJs"; true
-  }
-}
-var lang = 'en'
-const AudioScreen = ({ navigation }: any) => {
+let lang = 'en';
+
+function AudioScreen({ navigation }: RootStackScreenProps<'Audio'>) {
   const { i18n, t } = useTranslation(namespaces.pages.audio);
 
-  function handleClick(lang: string) {
-    i18n.changeLanguage(lang);
+  function handleClick(newLang: string) {
+    i18n.changeLanguage(newLang);
   }
   function toggle(lg: string) {
-    lang = (lg == 'en' ? 'es' : 'en')
-    return lang
+    lang = lg === 'en' ? 'es' : 'en';
+    return lang;
   }
   return (
     <ViewContainer>
-      <Text style={globalStyles.h2}>{t("audio_feed")}</Text>
-      <RectButton 
-      text="toggle"
-      {...t("buttons.ok", { ns: namespaces.pages.audio })}
-      onPress={() => handleClick(toggle(lang))}
+      <Text style={globalStyles.h2}>{t('audio_feed')}</Text>
+      <RectButton
```

Comment:
I'm getting typescript errors for missing props -- `textStyle` and `buttonStyle`. You could fill them in with `{{}}` for now